### ISSUE TITLE
 Happy New Year, i try but decrypt but...

### ISSUE THREAD

**Issue by captaingr** [themaddoctor/linux-mybook-tools#28]
Hello Doctor ;) 
I create usb boot ubuntu with 18.04.1 ver,  i run python and ubuntu say me I  have the new python, next step  run the pycrypto and all its ok, i run all step from "Mounting encrypted WD disks in linux.pdf" , i change the /dev/sdc with /dev/sda because sdc found my usb boot stick  sda its wd internal disk.
i try run all step but i have error in one step : ubuntu@ubuntu:~/wd$ dd if=kb.bin bs=8 skip2 count=5 of=edek.bin 
dd: kb.bin: cannot skip to specified offset
0+0 records in
0+0 records out
0 bytes copied, 0.000389845 s, 0.0 kb/s

do you have any idea?
my disk it 1TB , i am windows admin but i have no experience in linux.
the disc belongs to my brother and contains foto family :(

**Comment by themaddoctor** [comment]
Fist of all, "skip2" is not part of the instructions.

**Comment by captaingr** [comment]
Yes you are right . I have made a mistake in my message. I have written ".....skip=2 ...." and i read after "dd: kb.bin: cannot skip to specified offset"

**Comment by themaddoctor** [comment]
You must have a permissions problem. Try putting "sudo " in front of the command.

**Comment by captaingr** [comment]
You are amazing,  Friday it's starting to be copied the files  today Monday in 4 hours I think it's over !!! 
Ubundu it's magic , it's time to start learning.
I want to send you a gift from Greece send me your e-mail .
